src/_posts/2020-12-04-cass_changelog_2.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Apache Cassandra Changelog #2 | December 2020"
+date:   2020-12-01 x
+author: the Apache Cassandra Community
+categories: blog
+---
+![Apache Cassandra Changelog Header](img/changelog_header.jpg "image_tooltip")
```

Comment:
There are two things that need to change with this line.

1. This image block needs to be moved under the "_Our monthly roundup of key activities and knowledge to keep the community informed._" line. The reason is the image will appear in the blog summary page.
1. The URL is needs a leading `/`. It should be:
```
/img/changelog_header.jpg "image_tooltip"
```

Currently, this line results in the blog summary page rendering as follows.

<img width="884" alt="Blog_summary" src="https://user-images.githubusercontent.com/14037529/101109744-aedaca80-362b-11eb-9e2c-f4e2fbb649cc.png">